gym/lib/gym/detect_values.rb

```diff
@@ -1,3 +1,4 @@
+require 'cfpropertylist'
```

Comment:
Looks like that gem is actually not a direct dependency of fastlane, but of `xcodeprojc`, we should add it to `fastlane.gemspec` to be more explicit in case the dependency is dropped one day.